public/app/plugins/datasource/tempo/metricsSummary.ts

```diff
@@ -101,7 +101,7 @@ export function createTableFrameFromMetricsSummaryQuery(
       field.values.push(trace[field.name]);
     }
   }
-  frame = sortDataFrame(frame, 0);
```

Comment:
This sorts the `<no value>` so it's displayed closer towards the bottom of the table. I'm at two minds about this but thought it would be more beneficial for the user to see rows with values first (that's if any rows at all are missing values).